Formula/jolie.rb

```diff
@@ -0,0 +1,54 @@
+class Jolie < Formula
+  desc "The Jolie Language Interpreter"
+  homepage "https://www.jolie-lang.org/"
+  url "https://github.com/jolie/jolie/releases/download/v1.9.0/jolie-1.9.0.jar"
+  sha256 "1510ed7f114909eb79670462571ab0734a5b01e57d26da6fd1cf9ef6c67eff6e"
+
+  depends_on "openjdk"
+
+  def install
+    system "java", "-jar", "jolie-#{version}.jar", "--jolie-home", libexec, "--jolie-launchers", include
+    Dir.each_child(include) do |filename|
```

Comment:
Please use the wrapper function I linked you earlier.

Reply:
Got it, fixed :)